.rubocop.yml

```diff
@@ -43,4 +43,10 @@ Metrics/BlockLength:
   Exclude:
     - 'config/routes.rb'
     - 'spec/**/*'
-    - 'config/environments/*'
\ No newline at end of file
+    - 'config/environments/*'
+
+Metrics/ClassLength:
+  Description: 'Avoid classes longer than 110 lines of code.'
+  Enabled: true
+  CountComments: false
+  Max: 110
```

Comment:
Issue model is bigger than 100 lines now... and it looks like when i will merge your last commit will be even bigger

But we can try to move some methods to concerns...